target/linux/airoha/files/drivers/pinctrl/pinctrl-airoha.c

```diff
@@ -1681,7 +1895,8 @@ static int airoha_pinmux_set_mux(struct pinctrl_dev *pctrl_dev,
 		for (j = 0; j < group->num_regs; j++)
 			airoha_pinctrl_mux_set(pinctrl, base,
 					       group->regs[j].offset,
```

Comment:
@merbanan I think here we should just use `airoha_pinctrl_rmw()`

Reply:
Ack